ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -375,6 +376,12 @@ func (r *RayJobReconciler) getSubmitterTemplate(rayJobInstance *rayv1alpha1.RayJ
 		r.Log.Info("User-provided command is used", "command", submitterTemplate.Spec.Containers[0].Command)
 	}
 
+	// Set PYTHONUNBUFFERED=1 for real-time logging
+	submitterTemplate.Spec.Containers[0].Env = append(submitterTemplate.Spec.Containers[0].Env, v1.EnvVar{
```

Comment:
Nit: What will happen if users specify `PYTHONUNBUFFERED` on their own? I am not sure whether defining the same env variable twice in a container will throw an error message or override the first value.

Reply:
It seems the second definition will override the first one: https://www.baeldung.com/linux/kubernetes-pod-environment-variables